README.md

```diff
@@ -15,9 +15,8 @@ It supports up to 50 machines.
 To use local storage, each machine requires a local disk.
 To use distributed storage, at least three additional disks (not only partitions) for use by Ceph are required, and these disks must be on at least three different machines.
 
-<p align="center" width="100%">
-    <img alt="MicroCloud logo" width="33%" src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_236,h_214/https://assets.ubuntu.com/v1/904e5156-LXD+illustration+2.svg">
-</p>
+![MicroCloud logo](doc/images/microcloud_logo_dark.svg#gh-dark-mode-only)
+![MicroCloud logo](doc/images/microcloud_logo_light.svg#gh-light-mode-only)
```

Comment:
I guess we have to apply some scaling, otherwise the image takes up the entire width, see https://github.com/canonical/microcloud/blob/48df987a28d309d0491e4c80f375153d7dbd57b5/README.md.

Would something like this work?

```html
<p align="center" width="100%">
    <img alt="MicroCloud logo" width="33%" src="doc/images/microcloud_logo_dark.svg#gh-dark-mode-only">
    <img alt="MicroCloud logo" width="33%" src="doc/images/microcloud_logo_light.svg#gh-light-mode-only">
</p>
```

Reply:
> What is the reason for displaying both logos here? Shouldn't it be only one of them?

only one is shown at a time depending on whether you use dark or light mode.

Reply:
Did some tests, if we use `10%` scaling it's almost the same before.

Now I am asking myself why the original picture was even placed there in between the lines?
Should we maybe place the new logo right at the top?

![image](https://github.com/user-attachments/assets/246dcaf2-4761-41d8-92c0-fc56aee8a03a)

Reply:
Or let it float on the left like the rest of the page:

![Screenshot from 2024-11-08 14-17-19](https://github.com/user-attachments/assets/be2905dd-642e-413f-87fa-0bf6f864165e)

Reply:
This one looks niftier ;)

Reply:
Looks good, thanks.